AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 ||production-cmp.isgprivacy.cbsi.com/cps/shamanNotifier.js$domain=last.fm
+neterou.com##.cookie-class
```

Comment:
I don't see it in html and scripts.

Reply:
@Alex-302 Visit the site. `neterou.com`. I added it additionally  because I found it in `neterou.com`

Reply:
Removed.